.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: go
 go:
   - 1.5.1
-branches:
-  only:
-    - master
 services:
   - docker
 sudo: required
 script:
-  - make bootstrap test
+  - make test docker-build
```

Comment:
I think `bootstrap` still needs to be in, as well as the `build` target, since that creates the binary that the `Dockerfile` copies into the image